articles/databox-online/azure-stack-edge-gpu-activation-key-vault.md

```diff
@@ -41,6 +41,8 @@ A key vault is created for Azure Stack Edge resource during the process of activ
 
     ![MSI created during Azure Stack Edge resource creation](media/azure-stack-edge-gpu-deploy-prep/create-resource-8.png)
 
+- You can browse to the Azure Key Vault by clicking on the key vault name under properties within the Azure Stack Edge resource. 
```

Comment:
Could you please change to: To browse to the Azure Key Vault, go to the *Properties* in your Azure Stack Edge resource and select the key vault name.